libsolidity/codegen/YulUtilFunctions.h

```diff
@@ -90,10 +90,26 @@ class YulUtilFunctions
 	/// signature: (slot) ->
 	std::string copyLiteralToStorageFunction(std::string const& _literal);
 
+	/// @returns statements to revert with an error.
+	/// Generates code to revert with an error. The error arguments are assumed to
+	/// be already evaluated and available in local IRVariables, but not yet
+	/// converted.
+	std::string revertWithError(
+		std::string const& _signature,
+		std::vector<Type const*> const& _parameterTypes,
+		std::vector<ASTPointer<Expression const>> const& _errorArguments,
+		std::string const& _pos = {},
+		std::string const& _end = {}
```

Comment:
These names are a bit misleading. You should at least give a hint that these are variable names:

```suggestion
		std::string const& _posVar = {},
		std::string const& _endVar = {}
```